src/boot/efi/meson.build

```diff
@@ -174,6 +174,9 @@ efi_c_ld_args = [
         '-T' + elf2efi_lds,
 ]
 
+# CentOS 8 does not like this
```

Comment:
Can we please get a real comment here? It's not even clear what "this" is, the option?

Reply:
updated